src/dtflw/input_table.py

```diff
@@ -40,4 +61,4 @@ def validate(self, strict: bool = False):
             No used here but added for compatiblity with output_table.validate.
         """
         if self.needs_eval():
-            raise Exception("Required input not found.")
+            raise Exception(f"Required input not found.")
```

Comment:
Why the format here?

Reply:
@dr-BEat Added mistakenly I guess the PR is not fully ready yet, I will keep pushing changes.